packages/jest/src/generators/jest-project/files/jest.config.js__tmpl__

```diff
@@ -21,6 +16,9 @@ module.exports = {
     moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],<% } else { %>
     moduleFileExtensions: ['ts', 'js', 'html'],<% } %><% } %>
   coverageDirectory: '<%= offsetFromRoot %>coverage/<%= projectRoot %>'<% if(!skipSerializers) { %>,
+  transform: {
+    '^.+\\.(ts|js|html)$': 'jest-preset-angular'
+  },
```

Comment:
we'd have to wrap this logic in a same type of if as it was before:

```suggestion
<%if (setupFile === 'angular') { %>
   transform: {
     '^.+\\.(ts|js|html)$': 'jest-preset-angular'
   },
<% } %>
```

Also watch out for the previous `if` (`<% if(!skipSerializers) {`). This whole template is tricky so make sure to double check all the options when testing

Reply:
Resolved by splitting out the angular templates into files-angular.